hkmc2/shared/src/test/mlscript/backlog/Lifter.mls

```diff
@@ -160,7 +160,13 @@ fun foo(x, y) =
 //│ ═══[WARNING] Modules are not yet lifted.
 //│ ╔══[COMPILATION ERROR] No definition found in scope for 'M'
 //│ ║  l.154: 	  module M with
-//│ ╙──       	         ^
+//│ ║         	         ^^^^^^
+//│ ║  l.155: 	    fun foo() =
+//│ ║         	^^^^^^^^^^^^^^^
+//│ ║  l.156: 	      set y = 2
+//│ ║         	^^^^^^^^^^^^^^^
+//│ ║  l.157: 	      x + y
+//│ ╙──       	^^^^^^^^^^^
```

Comment:
Do you think the location is too... long? 🤔

Reply:
Yeah, in other prototypes we had logic to abbreviate long locations. We should port that over at some point.